makefile

```
@@ -1,36 +1,18 @@
-	@echo ""
-	@python3 gen.py
+# all: assets
+# 	@echo ""
+# 	@python3 gen.py
 
-.PHONY: assets docs
-
-node_modules:
-	npm ci
+.PHONY: astro
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)